orbital-mechanics/Lecture7/Lecture7.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<p>There are two similar issues here (and probably in the rest of the notebook:</p><ol><li> The first is with using $$ to write minimum propellant. Remove that as it looks weird in html. Instead, use boldface for markdown; e.g., this can be done using **minimum propellant**. If you want italics, you write __minimum propellant__.</li><li>For the equation you have written above, it looks good here on NBReview but it looks weird in the online textbook here: http://www.angadhn.com/SpacecraftDynamics/orbital-mechanics/Lecture7/Lecture7.html Solution is to start all new equations with one empty line above it (and probably below it.</li><li>In fact, I would like Equation numbering to show up. So what you can do (with chatGPT's help, of course) is to rewrite equations that are not inline in the following way:</li></ol>

<pre class="ql-syntax" spellcheck="false"><code></code>`{math}
:label: L7_1
\ddot {\bf r} = ...
<code></code>`
</pre><p>The label numbering format should should start with L7 in your case, then underscore, then the number of the equation. So, just give ChatGPT the entire ipynb and the above sample, it should be able to do the rest. The label is useful so we can refer to it in the online text, somewhere. Makes sense?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angadhn/SpacecraftDynamics/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Regarding item 3 above, I just adapted from this site's instructions (which is how we are creating an online book): https://jupyterbook.org/en/stable/content/math.html#math-blocks

Reply:
<p>I'm following those instructions and they are the same offered by ChatGPT. Though I can't render it in any way. I've tried it on VSCode with the enhanced markdown extension, I've also tried converting to html and here on nb notebook. </p>

Reply:
With your current setup; this is what the list looks like: http://www.angadhn.com/SpacecraftDynamics/orbital-mechanics/Lecture7/Lecture7.html

This is one of the examples where ChatGPT is wrong and I am right. So just copy paste (for now) and I will show you that it works on the final website.

Reply:
<p>Oh I see. </p><p> </p><p>P.S. how did that happen? I haven't created a pull on that commit yet</p>

Reply:
I do some magic ;)